### ISSUE TITLE
Unable to Add aria-label to Nav Links in Menu Widget

### ISSUE THREAD

**Issue by jared-leddy** [soflyy/oxygen-bugs-and-features#3392]
## **Describe the bug**

Using the built-in Menu widget, we need the option to add an `aria-label` to each link. With this widget, you have the option to add a custom attribute, but this would be added to the `<nav>` tag. We don't have the ability to directly add the label to each link. Which will continue to throw an error on ADA reports. 

## **A link to a Sandbox site where the bug has been reproduced https://oxygenbuilder.com/try? (REQUIRED)**

https://oxygen-qjz5kzy22b4tl.oxygen-demo.qsandbox.org/

## ADA Reference

https://dequeuniversity.com/rules/axe/3.5/link-name?application=axeAPI
